adoc/chapters/programming_interface.adoc

```diff
@@ -20898,7 +20898,7 @@ a@
 ----
 genint ilogb (genfloat x)
 ----
-   a@ Return the exponent as an integer value.
+   a@ Compute the integral part of logr (latexmath:[|x|]) and return the result as an integer, where r is the value returned by std::numeric_limits<genfloat>::radix.
```

Comment:
```suggestion
   a@ Compute the integral part of logr (latexmath:[|x|]) and return the result as an integer, where r is the value returned by [code]#std::numeric_limits<genfloat>::radix#.
```

Probably, this should be formatted as code.  Same for the description of `logb`.

Reply:
Thanks for the input. I've used the code formatting now.